docs/docs/data-structures/tree/schema-definition.mdx

```diff
@@ -54,8 +54,9 @@ class Note extends sf.object("Note", {
 
 For the remainder of this article, we use the inline style.
 
-You can add fields, properties, and methods like any TypeScript class including methods that wrap one or more methods in the <PackageLink packageName="tree">SharedTree API docs</PackageLink>.
```

Comment:
The latter part of this sentence was confusing and doesn't seem necessary. Rather than try to figure out the appropriate API to link to, I opted to just remove that part of the sentence.